platforms/aws/giantnetes/variables.tf

```diff
@@ -24,6 +24,12 @@ variable "worker_count" {
   default     = "4"
 }
 
+variable "expiration_days" {
```

Comment:
You still need to pass this variable to s3 module

https://github.com/giantswarm/giantnetes-terraform/blob/master/platforms/aws/giantnetes/main.tf#L55-L60